docs/topics/migrations.txt

```diff
@@ -218,6 +218,28 @@ Apps without migrations must not have relations (``ForeignKey``,
 ``ManyToManyField``, etc.) to apps with migrations. Sometimes it may work, but
 it's not supported.
 
+Swappable dependencies
+----------------------
+
+.. function:: django.db.migrations.swappable_dependency(value)
+
+The ``swappable_dependency()`` function is used in migration files to specify
+a swappable dependency for a particular migration operation. It is not
+directly related to swapping models but rather influences how migrations
+handle the dependency resolution process.
```

Comment:
```suggestion
The ``swappable_dependency()`` function is used by migration operations to declare "swappable" dependencies such as swappable user models.
```
Suggestion: be direct and to-the-point:
 - used by migration operations
 - clarify what a "swappable dependency" is with the classic example: user models
 - avoid repetition "migration files … migration operation"; simply referring to migration operation is enough to indicate its usage

Additionally: I'm not sure the second sentence is useful. It appears contradictory as the very next paragraph talks about how it's commonly used with swappable models.